modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,49 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling mistakes.
+:keywords: autocorrect, capitalization, spell, spelling, spellcheck
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tiertwo
```

Comment:
```suggestion
:pluginminimumplan: tierthree
```

I saw @MitchCTiny changed this in footnotes. He was correct there because it's poorly named... `tierone` is Community, `tiertwo` is Essential and and `tierthree` is Professional.

Reply:
`tiertwo` was a placeholder until I found out the correct plan, so yes autocorrect is `tierthree` too, while merge-tags can remain `tiertwo`